<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>272</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>475</width>
    <height>272</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>475</width>
    <height>272</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pass Word Generator</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>1024</width>
     <height>768</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="bUseCaps">
         <property name="text">
          <string>Use capital letters</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="bUseNumbers">
         <property name="text">
          <string>Use numbers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="bUseSpecialChars">
         <property name="text">
          <string>Use special characters</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="leSpecialChars">
           <property name="text">
            <string>!&quot;#$%&amp;'()*+,-./:;&lt;=&gt;?@[\]^_`{|}~</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnResetSpecialChars">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Min Length:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lePassWordMinLength">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>32</string>
           </property>
           <property name="maxLength">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Max Length:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lePassWordMaxLength">
           <property name="text">
            <string>48</string>
           </property>
           <property name="maxLength">
            <number>2</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btnGenPassWord">
         <property name="text">
          <string>Generate Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lePassWordResult"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
